yugabyte/run-jepsen.py

```diff
@@ -258,6 +258,8 @@ def send_report_to_reportportal(
         logging.error(f"Code: {response.status_code} Text: {response.text}")
         return False
 
+    return True
```

Comment:
Instead of returning True here, should not we remove else statement and put return in if clause. That makes more sense. Because we have an if clause but no return statement there.

```
if response.status_code == 200:
        logging.info(f"Successfully updated attributes for launch {launch_id}")
        return True
logging.error(f"Could not update attributes for launch {launch_id}")
logging.error(f"Code: {response.status_code} Text: {response.text}")
return False
```